Makefile

```diff
@@ -293,7 +295,13 @@ clean:
 	@-$(RM_RF) ./*/bin ./*/obj
 	@-$(RM_RF) ./thirdparty/download
 
-distclean: clean
+distclean: clean cleanversion
+	@-$(RM_F) Changelog.md
+	@-$(RM_F) *.tar.bz2
+
+dist: version
+	@-$(CURL) -s -L -O https://raw.githubusercontent.com/wiki/OpenRA/OpenRA/Changelog.md
```

Comment:
The latest version of Changelog.md corresponds to the head of the current prep branch, which in general will *not* be the same as the version being compiled here.  It would be fine to do this from the packaging scripts (because they are called in the one context where this assumption is reasonable), but not really from the Makefile.

I can suggest two solutions here:
* Don't ship a changelog in the tarball
* Move this into the packaging scripts where we can safely(ish) assume the contents to be correct.

Reply:
It shouldn't matter as `make dist` is just for us and will be invoked during the release time frame where those line up.

Reply:
I'd agree with that if it were done exclusively in the packaging scripts, but by putting it in the general makefile we are explicitly advertising it as a general feature that anyone can use.

Reply:
Removed it then. I just read on the GNU standards that every distribution tarball should come with installation instructions and a changelog.

Reply:
We could still fulfill that by doing the packaging in the packaging scripts.